src/system/ThemeModule/Resources/config/theme_engine_event_listeners.yml

```diff
@@ -60,7 +60,7 @@ services:
           - "@zikula_extensions_module.api.variable"
           - "@zikula_users_module.current_user"
           - "@templating.engine.twig"
-          - "@zikula_core.common.theme.themevars"
+          - "@zikula_core.common.theme.pagevars"
```

Comment:
This fixes a bug which caused that polyfills never went into the `ZConfig` array.